### PR TITLE
UI to show the progression status when cleanup/failover/relocate operations remain stuck

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Show progression status in addition to status
for the scenarios where cleanup or the
failover/relocate operations remain stuck
https://bugzilla.redhat.com/show_bug.cgi?id=2189999

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
Screenshot from 2023-08-08 00-36-10

**Comment by TimothyAsirJeyasing** [comment]
@SanjalKatiyar , @GowthamShanmugam Please review

**Comment by SanjalKatiyar** [comment]
update the PR title correctly as per the guidelines...

**Comment by TimothyAsirJeyasing** [comment]
/retitle UI to show the progression status when cleanup/failover/relocate operations remain stuck

**Comment by SanjalKatiyar** [comment]
> /retitle UI to show the progression status when cleanup/failover/relocate operations remain stuck

add bug to the description (plz check the shared guidelines doc and follow all steps correctly)...

File: packages/mco/utils/disaster-recovery.tsx [hunk 1]
```diff
@@ -439,7 +441,9 @@ export const getDRStatus = ({
     case DRPC_STATUS.Relocating:
     case DRPC_STATUS.FailingOver:
       return {
-        text: customText || currentStatus,
+        text:
+          (customText || currentStatus) +
+          (progression ? ` ${progression}` : ''),
```
Comment (GowthamShanmugam) [inline packages/mco/utils/disaster-recovery.tsx:446]: 1. Is this UI change discussed with UX?
2. It looks like the Main status For all operations like Failover and Relocate will be replaced by all of these sub statuses: https://github.com/RamenDR/ramen/blob/main/api/v1alpha1/drplacementcontrol_types.go#L77
3. Also, What about the subscription type? this UI covers only the Application set.
4. Is any changes required in the dashboard also?

File: packages/mco/components/modals/app-data-policies-status/argo-application-set.tsx [hunk 2]
```diff
@@ -43,6 +43,7 @@ export const ArogoApplicationSetStatus: React.FC<ArogoApplicationSetStatusProps>
         const drpc = drResource?.drPlacementControls?.[0];
         const drPolicy = drResource?.drPolicy;
         const status = drpc?.status?.phase as DRPC_STATUS;
+        const progression = drpc?.status?.progression;
```
Comment (GowthamShanmugam) [inline packages/mco/components/modals/app-data-policies-status/argo-application-set.tsx:46]: Add progression status under DRPlacementControlKind

**Pull request closed without merging**